CMakeLists.txt

```diff
@@ -39,10 +39,18 @@ else()
   common_package(DNSSD)
 endif()
 common_package(Boost 1.51 COMPONENTS unit_test_framework)
+common_package(Qt5Core)
+common_package(Qt5Widgets)
 common_package(Threads REQUIRED)
 common_package_post()
 
+if(Qt5Core_FOUND)
+  set(SERVUS_DEPENDENT_LIBRARIES Qt5Core)
```

Comment:
I don't remember how this DEPENDENT_LIBRARIES work, but I suppose it is per project and not per library then.